src/newLaunch/baseStage.scss

```diff
@@ -864,3 +864,26 @@
     }
   }
 }
+
+@media screen and (max-width: 690px) {
+  .distribRow {
+    position: relative;
+    margin-top: 25px;
```

Comment:
rather than margin-top, can you please configure the flex that is defined by `.distribRow`?  If that is not feasible, switch to grid.